<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientRepairsWindow</class>
 <widget class="QWidget" name="ClientRepairsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="machine_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Выберите станок</string>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="repair_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Выберите услугу</string>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="go_area_button">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>390</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Личный кабинет</string>
   </property>
  </widget>
  <widget class="QLabel" name="list_reapirs_label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>471</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Список ремонтных работ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="order_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Заказать</string>
   </property>
  </widget>
  <widget class="QComboBox" name="repair_combo_box">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>160</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>Услуга</string>
   </property>
  </widget>
  <widget class="QComboBox" name="machine_combo_box">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>90</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>Станок</string>
   </property>
  </widget>
  <widget class="QLabel" name="execution_time_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Время выполнения</string>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="execution_cost_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Стоимость выполнения</string>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="execution_time_line_edit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>230</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="execution_cost_line_edit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>300</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
